### PR TITLE
[MGDOBR-849] Prapare CI for testing of Ansible Tower action

### PR DESCRIPTION
**Pull request by jiripetrlik**
[MGDOBR-849](https://issues.redhat.com/browse/MGDOBR-849)

Add environment variables for Ansible instance to CI.

### CODE REVIEW

File: .github/workflows/Manager-CI.yaml [hunk 1]
```diff
@@ -158,6 +158,9 @@ jobs:
           SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
       - name: Run integration tests
         env:
+          ANSIBLE_ENDPOINT: ${{ secrets.ANSIBLE_ENDPOINT }}
+          ANSIBLE_USERNAME: ${{ secrets.ANSIBLE_USERNAME }}
+          ANSIBLE_PASSWORD: ${{ secrets.ANSIBLE_PASSWORD }}
```
Comment (sutaakar) [inline .github/workflows/Manager-CI.yaml:163]: Do we have Ansible available for upstream testing?

Comment (robpblake) [reply .github/workflows/Manager-CI.yaml:163]: @jiripetrlik What about for local dev? Does the team need to know about the update for these tests?

Comment (jiripetrlik) [reply .github/workflows/Manager-CI.yaml:163]: @robpblake Values for these environment variables were added to our "OpenBridge - Environment Details" spreadsheet.

**Comment by robpblake** [comment]
@jiripetrlik Given the new focus on v2, I would suggest that we close this PR now and don't progress with this?

cheers,

Rob

**Comment by jiripetrlik** [comment]
@robpblake 
Understand. I'm closing the issue.

**Pull request closed without merging**